src/validator/IndexScanValidator.cpp

```diff
@@ -46,6 +58,18 @@ Status IndexScanValidator::prepareFrom() {
 Status IndexScanValidator::prepareYield() {
     auto *sentence = static_cast<const LookupSentence *>(sentence_);
     returnCols_ = std::make_unique<std::vector<std::string>>();
+    // always return
+    if (isEdge_) {
+        returnCols_->emplace_back(kSrc);
+        colNames_.emplace_back(kSrcVID);
+        returnCols_->emplace_back(kDst);
+        colNames_.emplace_back(kDstVID);
+        returnCols_->emplace_back(kRank);
+        colNames_.emplace_back(kRanking);
+    } else {
+        returnCols_->emplace_back(kVid);
+        colNames_.emplace_back(kVertexID);
+    }
```

Comment:
Why save two copies? IIRR，Other PR is already dealing with this issue.

Reply:
They are different see the line 112

Reply:
> They are different

Yes, I see a difference, but I think one copy is enough

Reply:
Is there conflict with other PR : 
https://github.com/vesoft-inc/nebula-storage/pull/224

Reply:
Won't I test in this PR.

Reply:
> Won't I test in this PR.

got.